ksql-engine/src/main/java/io/confluent/ksql/services/TopicValidationUtil.java

```diff
@@ -19,13 +19,13 @@
 import io.confluent.ksql.topic.TopicProperties;
 import org.apache.kafka.clients.admin.TopicDescription;
 
-final class TopicValidationUtil {
+public final class TopicValidationUtil {
```

Comment:
I think this would be better duplicated.  Only one method is needed, in one place. I'd just inline that method.

Reply:
Done